diff --git a/build/src/start/server/metro/MetroBundlerDevServer.js b/build/src/start/server/metro/MetroBundlerDevServer.js
index a8dbeb2d8d2f1263a75805a10b92c1faa775490c..0e05082088d37992cdadbc72c253d23a0905512c 100644
--- a/build/src/start/server/metro/MetroBundlerDevServer.js
+++ b/build/src/start/server/metro/MetroBundlerDevServer.js
@@ -784,7 +784,9 @@ class MetroBundlerDevServer extends _BundlerDevServer.BundlerDevServer {
         };
         // https://github.com/facebook/metro/blob/2405f2f6c37a1b641cc379b9c733b1eff0c1c2a1/packages/metro/src/lib/parseOptionsFromUrl.js#L55-L87
         if (!opts.mainModuleName.startsWith('/') && !_path().default.isAbsolute(opts.mainModuleName)) {
-            opts.mainModuleName = './' + opts.mainModuleName;
+            // FIXME: Hard-coded due to not working as expected in monorepo.
+            opts.mainModuleName = "./example";
+            // opts.mainModuleName = './' + opts.mainModuleName;
         }
         const output = await this.metroLoadModuleContents(opts.mainModuleName, opts, extraOptions);
         return {
